docs/glossary.md

```diff
@@ -8,6 +8,10 @@ sidebar_position: 4
 
 Decentralized application.
 
+## Endpoint
+
+The cross-chain operation entry. An endpoint is a deployed smart contract which is used by Dapps to do cross-chain operations. 
```

Comment:
Could you please give an example here? Or if these glossaries are all related to the SDK, we might put it under SDK folder?

Reply:
What kind of example?

Reply:
A contract defined and deployed by the Dapp developer. 
```
contract XxxEndpoint is MessageEndpoint {...}
```
```
contract XxxDapp {
   XxxEndpoint endpoint;
   
   function foo() {
       endpoint.remoteExecute();
   }
}
```

Reply:
move Glossary to the end